### ISSUE TITLE
implement sounds

### ISSUE THREAD

**Issue by aspizu** [aspizu/gobomatic#5]

**Comment by aspizu** [comment]
need a way to get the sample rate and samples count from the audio file
this needs to support all the filetypes that scratch supports.

**Comment by aspizu** [comment]
would it be fine if it only supported wav files ?

**Comment by themysticsavages** [comment]
yeah sure, most projects use wav, the entire default sounds library is wav (i think), and its a good starting point
